hooks/pre/30-install_redis_scl.rb

```diff
@@ -0,0 +1,10 @@
+# Working around https://tickets.puppetlabs.com/browse/PUP-2169
+if facts[:os][:release][:major] == '7'
```

Comment:
Please also check if this is the Red Hat family
```suggestion
if facts[:os][:family] == 'RedHat' && facts[:os][:release][:major] == '7'
```